src/Shared/RedisUtility.cs

```diff
@@ -5,13 +5,24 @@
 
 using System;
 using System.Collections.Generic;
+using System.Configuration;
 using System.IO;
 using System.Runtime.Serialization.Formatters.Binary;
 
 namespace Microsoft.Web.Redis
 {
     internal static class RedisUtility
     {
+        internal static ISerializer Serializer;
+
+        static RedisUtility()
+        {
+            var serializerType = ConfigurationManager.AppSettings["RedisSerializerType"];
```

Comment:
I think that to make that change viable we'll to convert the RedisUtility static class to a non-static class in order to inject the provider configuration.
Do you agree?